service/controller/aws/v4/resource_set.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/giantswarm/cluster-operator/service/controller/aws/v4/resource/awsconfig"
 )
 
+const (
+	// Provider is used to enable provider specific features.
+	Provider = "aws"
```

Comment:
Provider is set via constants in the resource set. This works I'm open to a more elegant solution if there is one.

Reply:
As I noted above, I need to do similar thing for `certconfigs` resource and I created entries to `label` package.